speech-to-speech/README.md

```diff
@@ -0,0 +1,14 @@
+# Cloud Speech to Speech Translation iOS Sample
+
+This app demonstrates how to make gRPC connections to the(API Integration):
+1. Make streaming gRPC connections to the [Cloud Speech API](https://cloud.google.com/speech/) to the recognize speech audio. Receiving Transcript.
+2. Send Transcript to [Translate API](https://cloud.google.com/translate/)
+3. Send Translated text to [Text-to-Speech API](https://cloud.google.com/text-to-speech/)
+4. [Get list of voices](https://cloud.google.com/text-to-speech/docs/list-voices)
+
+Check out app's README for full getting started instructions.
+
+
+## Contents
+
+### [SpeechToSpeech](SpeechToSpeech)
```

Comment:
Could probably delete this whole file, since it's covered in the next folders README already.

Reply:
Removed.